docs/spec/lcd/api.md

```diff
@@ -0,0 +1,847 @@
+
+```
+{
+    "error": "",
+    "code":200,
+    "result": {
+     {
+         "atom": 1000,
+         "photon": 500,
+         "ether": 20
+     }
+    },
+    "rest api": "2.0"
+}
+```
+
+* The above command returns JSON structured like this if fails:
+
+```
+{
+    "error": "Invalid account",
+    "code":500,
+    "result": {},
+    "rest api": "2.0"
+}
+```
+
+
+### /create_transfer - POST
+
+url: /create_transfer, Method: **POST**
+
+Functionality: transfer asset
+
+Parameters:
+
+| Parameter | Type   | Default | Required | Description                 |
+| --------- | ------ | ------- | -------- | --------------------------- |
+| from  | string | null    | true     | address from                |
+| to  | string | null    | true     | address want to send to     |
+| amount  | int    | null    | true     | amount of the token         |
+| denomonation  | string | null    | true     | denomonation of the token   |
+
+
+* The above command returns JSON structured like this if success:
+
+```
+{
+    "error": "",
+    "code":200,
+    "result": {
+     ""transaction": "[]bytes of a valid transaction bytes to be signed for that zone"
+    },
+    "rest api": "2.0"
+}
+```
+
+* The above command returns JSON structured like this if fails:
+
+```
+{
+    "error": "Insufficient Funds",
+    "code":500,
+    "result": {},
+    "rest api": "2.0"
+}
+```
+
+
+### /signed_transfer - POST
+
+url: /signed_transfer, Method: POST
+
+Functionality: transfer asset
+
+Parameters:
+
+| Parameter       | Type   | Default | Required | Description            |
+| ------------    | ------ | ------- | -------- | ----------------------------------------------- |
+| signed_transfer | []byte | null    | true     | bytes of a valid transaction and it's signature |
+
+* The above command returns JSON structured like this if success:
+
+```
+{
+    "error": "",
+    "code":200,
+    "result": {
+     "tx_hash": ""
+    },
+    "rest api": "2.0"
+}
+```
+
+* The above command returns JSON structured like this if fails:
+
+```
+{
+    "error": "Invalid Signature",
+    "code":500,
+    "result": {},
+    "rest api": "2.0"
+}
+```
+
+
+## ICS21 - StakingAPI
+
+The StakingAPI exposes all functionality needed to stake tokens. It is exposed by the staking
+module.
+
+
+### /stake/{delegator}/bonding_status/{validator} - GET
+
+url: /stake/{delegator}/bonding_status/{validator}, Method: GET
+
+Functionality: get delegator information and verify returned proof
+
+Parameters: null
+
+* The above command returns JSON structured like this if success:
+
+```
+{
+    "error": "",
+    "code":200,
+    "result": {
+     "validator":address1
+     "delegator":address2
+     "liability": 1
+    },
+    "rest api": "2.0"
+}
+```
+
+* The above command returns JSON structured like this if fails:
+
+```
+{
+    "error": "cannot send asset to the account",
+    "code":500,
+    "result": {},
+    "rest api": "2.0"
+}
+```
+
+
+### /stake/validators - GET
```

Comment:
Should be GET `/stake/validatorsets/latest`.
1. validatorsets are not static so `/stake/validatorsets/latest` will return sth different over time.
2. GET `/stake/validators` implicates we are querying for individual validators which we are not.